### DISCUSSION TITLE
Framework記述中

### DISCUSSION THREAD

**Discussion by KokoroKagawa** [copilot-jp/project-sprint#381] [Development]
まずは横のPJSを書ききる。
縦のPJSはAdvanceとして書かれるのかもしれない。

[Framework第3稿](https://github.com/copilot-jp/project-sprint/commit/c972b893df1a190693f318e586b899b707393cc0) https://github.com/copilot-jp/project-sprint/issues/343

**Comment by motoi** [comment]
https://github.com/copilot-jp/project-sprint/issues/374
イシューでのやりとりはこちら

**Comment by motoi** [comment]
https://github.com/copilot-jp/project-sprint/blob/b7e07ea41c8837324d535c4f93240393b6bf71c8/JA/v3.4/framework.md#user-content-fn-1-549c9899aed689b181aeb11c460af097
に対するコメント

**Reply by motoi** [reply]
> プロジェクトの推進とは、プロジェクトチームが小さな成果を生み出しつづけることを通してこの最適化のための情報を集め、

この最適化、が何をさしているのかわかりづらそう

**Reply by Satoshi-Matsumoto** [reply]
> 成果の変容: 実験に取り組むことで、プロジェクト外部の環境に関する情報（環境情報）が少しずつ明らかになる。

これ感覚的に分かる人はそうだよねー、なんですが、実験に取り組むことで外部が、、、、ってちょっとわからない人いるかも？内部と外部の境界、みたいな話も必要？

⇒[こころ]「出力すると外部が分かる」はより丁寧に説明するべき。センスメイキング理論。イナクトメントの効果。
実験に取り組む＝内部を進めるものでもあるし、内部から外部への働きかけでもある。

**Reply by motoi** [reply]
> Project Sprintは、このフレームワークを補強したり理解しやすくしたりするための方法論や考え方を、Tutorialとして提供する。Tutorialは、プロジェクトの現場でProject Sprintを実践する際の手助けになるだろう。

チュートリアルはカタカナにしようか。。。

**Reply by motoi** [reply]
> Project Sprintにふれるに当たって

どのような概念か？みたいな感じかな。

**Reply by Satoshi-Matsumoto** [reply]
> 出力目標

これもう気分だけの話かもしれませんが、「ターゲット」とかにしたほうがテンションあがりません？

**Reply by motoi** [reply]
> 実験には、二つのインパクトがある。

実験による作成物、のほうがよいか

**Reply by Satoshi-Matsumoto** [reply]
> 出力に取り組むことで得られた環境情報が、成果の計画を最適化する材料になる

これが絶妙に頭にはいってこない。。。。出力すると環境情報を得られて成果にたどり着くまでに何をするかが最適化される、みたいなことですかね？「成果の計画」っていうのが分かりにくいのかも。

**Reply by Satoshi-Matsumoto** [reply]
> その根底にあるのはメンバー間の信頼であり、この信頼の構築に最も効果的なのは出力に取り組むこと、つまり作成物を生み出すことである。作成物への取り組みは、成果を実現したり環境情報を取得したりする手段であるのと同様に、役割を引き受けてチームへの貢献と誠実さを示すことによって信頼関係を醸成する手段でもある。他のメンバーの作成物を率直かつ公正に受けとめることもまた、相互の信頼の構築に大きく役立つ。

名言

**Reply by Satoshi-Matsumoto** [reply]
> 各メンバーが自律的に出力に取り組むためには、次のようなことが必要である。
> 実現すべき成果がプロジェクトチーム全員によって合意・納得され、環境の変化に応じた成果の最適化も都度共有されて認識が揃っていること

これは、実現すべき成果は最初からプロジェクトチーム全員で合意・納得されてないといけない、と言っているわけではないですよね？念のため。

**Reply by motoi** [reply]
成果と出力のところの1-5の部分はあんまり説明的な文章なしにしてシンプルに書いても良いのでは？とおもいました。チュートリアルぽい？

**Reply by Satoshi-Matsumoto** [reply]
> 重要なのは、チームメンバー全員が一時的に同一の環境に固定されてリアルタイムで会話をすることにより、素早く効率的な認識合わせと全員にとって納得感のある意思決定という、同期的な会議でしか生まれない効果を生むことだ。これを定期的・反復的に実施して各メンバーの作成物への取り組みを共有することにより、次のようなことが可能になる。

サトマツ最近「会議をすることで人柄とか癖とか知って距離が近くなる（オフラインだとベター）」みたいなすごく人間的な価値の重要性を感じるのですがそういうのってProject Sprintでは取り扱わないですか？

⇒[こころ]大事ではある。Project Sprint的か？フレームワークの範疇か？
情報量の増加。（チャット⇒オンラインミーティング⇒オフライン）
本が出たときのコラムに書きたい内容。
